sympy/geometry/ellipse.py

```diff
@@ -1268,6 +1268,45 @@ def vradius(self):
         return self.args[2]
 
 
+    def second_moment_of_area(self, p=None):
```

Comment:
I think you should change the polygon class to use `p=None` instead of `p='centroid`.

Reply:
@moorepants are talking about this pr #13939 (https://github.com/sympy/sympy/pull/13939/files/c7fe628b1b238f01f3ec61a2a3b44aa923147e55).
am I supposed change p='centroid' to None

Reply:
Yes. I think we should have consistent api's for this method on geometry objects and using `None` as a default when there is only one possible kwarg aligns more with common practice.

Reply:
And one other nitpick, would be to make it:

```python
def second_moment_of_are(self, point=None):
```